### PR TITLE
Add image related apis

### PR DESCRIPTION
**Pull request by ehgmsdk20**
Changes:
- upload image with buffer
- upload image with data url
- delete image

### CODE REVIEW

**Review by platfowner — APPROVED** [review]
LGTM
Thanks!

File: src/nft.ts [hunk 1]
```diff
@@ -184,4 +186,37 @@ export default class Nft extends AinftBase {
...
+      }
+    });
+  }
+
+  uploadAssetWithDataUrl({
+    appId,
+    filename,
+    dataUrl,
+    filePath
+  }: UploadAssetFromDataUrlParams): Promise<TransactionInput> {
```
Comment (woomurf) [inline src/nft.ts:212]: if user does not send transaction, image will be deleted?

File: src/nft.ts [hunk 2]
```diff
@@ -184,4 +186,37 @@ export default class Nft extends AinftBase {
...
+      appId, ...(filePath && {filePath})
+    }, {
+      asset: {
+        filename,
+        buffer
+      }
+    });
+  }
+
+  uploadAssetWithDataUrl({
```
Comment (woomurf) [inline src/nft.ts:207]: how about getTxBodyFor...?

File: src/nft.ts (hunk 1 above)
Comment (ehgmsdk20) [reply src/nft.ts]: This function does not return txBody, it returns asset url. i will fix it thanks!

File: src/nft.ts (hunk 2 above)
Comment (ehgmsdk20) [reply src/nft.ts:218]: This function does not return txBody, it returns asset url. i will fix it thanks!

**Pull request merged**